tests/helpers/streamlit_tests/test_streamlit_show_resources.py

```diff
@@ -152,13 +153,12 @@ def test_render_with_pipeline_with_different_pipeline_dirs():
     def dummy_render(pipeline: dlt.Pipeline) -> None:
         pass
 
-    old_args = sys.argv[:]
-    with pytest.raises(CannotRestorePipelineException):
-        sys.argv = [*base_args, "/run/dlt"]
+    with pytest.raises(CannotRestorePipelineException), mock.patch.object(
```

Comment:
This is a leftover improvement from streamlit to actually patch `sys.argv` instead of manually handing it.